.github/workflows/sycl_linux_build_and_test.yml

```diff
@@ -157,15 +185,128 @@ jobs:
         cmake --build $GITHUB_WORKSPACE/build --target utils/llvm-lit/install
         cmake --build $GITHUB_WORKSPACE/build --target install-clang-format
         cmake --build $GITHUB_WORKSPACE/build --target install-clang-tidy
+        cmake --build $GITHUB_WORKSPACE/build --target install-llvm-size
+        # TODO this should be resolved in CMakeLists.txt
+        cmake --build $GITHUB_WORKSPACE/build --target install-lld || echo "skipped"
 
-    - name: Pack
+    - name: Pack toolchain
       run: tar -cJf llvm_sycl.tar.xz -C $GITHUB_WORKSPACE/build/install .
-    - name: Upload artifacts
+    - name: Pack LIT
+      run: tar -cJf lit.tar.xz -C $GITHUB_WORKSPACE/src/llvm/utils/lit .
+    - name: Upload toolchain
       uses: actions/upload-artifact@v1
       with:
         name: sycl_linux_${{ fromJSON(needs.configure.outputs.params).build_artifact_suffix }}
         path: llvm_sycl.tar.xz
-    - name: Cleanup
-      if: always()
-      run: rm -rf $GITHUB_WORKSPACE/*
+    - name: Upload LIT
+      uses: actions/upload-artifact@v1
+      with:
+        name: sycl_lit_${{ fromJSON(needs.configure.outputs.params).build_artifact_suffix }}
+        path: lit.tar.xz
 
+  llvm_test_suite_l0_gen9:
+    name: L0 GEN9 Test Suite
+    needs: [build, configure]
```

Comment:
One caveat: I think `build` here means "build+lit", right?
I think it might be useful to let developers run llvm-test-suite even if some LIT tests fail.

Reply:
Good point, I'll think about it